### ISSUE TITLE
Performance for Pedestrian and Cyclist on validation set

### ISSUE THREAD

**Issue by kwea123** [traveller59/second.pytorch#51]
Hi, can you also provide the mAP for Pedestrian and Cyclist on the validation set? I want to make some comparison between different methods. Thank you!
